examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -136,6 +136,18 @@ tls:
   # The filename inside the secret is important - it _must_ be ca.pem
   existingCASecretName: ""
 
+updater:
+  enabled: false
+  # `updater.versionServer` is the URL of the version server the agent fetches
+  # the target version from.
+  versionServer: "https://update.gravitational.io/v1/"
+  # Release channel the agent subscribes to.
+  releaseChannel: "stable/cloud"
```

Comment:
TBH I'm not sure why we split versionServer and releaseChannel in 2 parameters. Self-hosted users would need to change both anyway and it's not very clear that "channel" is actually just a path on the version server. Wouldn't having just a single parameter make things simpler and clearer?

```yaml
versionServer: https://update.gravitational.io/v1/stable/cloud
```

Reply:
> TBH I'm not sure why we split versionServer and releaseChannel in 2 parameters. Self-hosted users would need to change both anyway and it's not very clear that "channel" is actually just a path on the version server. Wouldn't having just a single parameter make things simpler and clearer?

I suspect having a dedicated release channel will be very handy when we will want to migrate existing cloud users into automatic upgrades. We will face strong coupling issues:
- a tenant can be enrolled into automatic upgrades if the majority of their agents are automatically upgraded
- we can't bump the target version of the cloud/stable channel if it will break at least a tenant

To support a seamless transition I think we will have to create per-tenant release channels. In this scenario, it will be less awkward for users to only specify their tenant-specific release channel while still using the default versionServer.

Reply:
I don't think we'll want to maintain per-tenant versions (sounds like a huge burden) but I understand that this allows you to change either server URL or version file path independently.

I think it's fine, my main issue was that "release channel" is just quite confusing term for what effectively is just a path component of the URL. Can we at least update the comment explaining that the updater will basically just call `filepath.Join(versionServer, releaseChannel)` to fetch the version information?